index.html

```diff
@@ -0,0 +1,586 @@
+						of light-years across. &ldquo;It&rsquo;s often said that there is no
+						such thing as a free lunch,&rdquo; Dr. Guth likes to say, &ldquo;but the
+						universe might be the ultimate free lunch.&rdquo;</p>
+
+					<p>Make that free lunches. Most of the hundred or so models resulting from Dr.
+						Guth&rsquo;s original vision suggest that inflation, once started, is
+						eternal. Even as our own universe settled down to a comfortable homey
+						expansion, the rest of the cosmos will continue blowing up, spinning off
+						other bubbles endlessly, a concept known as the multiverse.</p>
+
+					<p>So the future of the cosmos is perhaps bright and fecund, but do not bother
+						asking about going any deeper into the past.</p>
+
+					<p>We might never know what happened before inflation, at the very beginning,
+						because inflation erases everything that came before it. All the chaos
+						and randomness of the primordial moment are swept away, forever out of
+						our view.</p>
+
+					<p>&ldquo;If you trace your cosmic roots,&rdquo; said Abraham Loeb, a
+						Harvard-Smithsonian astronomer who was not part of the team, &ldquo;you
+						wind up at inflation.&rdquo;</p>
+					<div class="content-foot">
+						<span class="top-text">A version of this article appears in print on March 17, 2014, on
+							Page A1 of the New York edition with the headline: Space Ripples
+							Reveal Big Bang’s Smoking Gun. Order Reprints | Today’s Paper |
+							Subscribe</span> <br>
+						<button>READ 615 COMMENTS</button>
+						<ul class="social-icons">
+							<li class="icons rounded"><i class="fab fa-facebook"></i>
+							</li>
+							<li class="icons rounded"><i class="fab fa-twitter"></i>
+							</li>
+							<li class="icons rounded"><i class="fas fa-envelope"></i>
+							</li>
+							<li class="icons rounded"><i class="fas fa-share"></i>
+						</ul>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="sidebars">
+			<div class="editors">
+
+				<span>Editor's Picks</span>
+				<br>
+				<hr>
+				<br>
+
+				<div><img src="http://www.tapis-chic.com/img/p/product_list_new/841-5079.jpg" alt="thumbnails">
+					<p id="desc1">Bigger, Saltier, Heavier: Fast Food Since 1996 in 3 Simple Charts </p>
+				</div>
+				<div><img src="http://www.freedigitalphotos.net/blog/wp-content/uploads/2013/11/archie.jpg"
+						alt="thumbnails">
+					<p> He Lives to Provoke. What HAppens When His Film Isn't Hated?</p>
+				</div>
+				<div><img
+						src="https://dyw7ncnq1en5l.cloudfront.net/optim/produits/16/20045/sony-hx400v_b3828afdd76c9011__450_400.jpg"
+						alt="thumbnails">
+					<p id="desc2"> There Are No Five Stages Of Grief</p>
+				</div>
+			</div>
+			<div class="science">
+				<span>Sign up for Science Times</span><br>
+				<span>We'll bring you stories that capture the wonders of the human body, nature and the cosmos.</span>
+				<br>
+				<button>SIGN UP</button>
+			</div>
+		</div>
+	</main>
+
+	<footer>
```

Comment:
footer has enormous size,due to floats, please clear the floats.

Reply:
We cleared the floats, and footer now has a normal size.